src/fhir.js

```diff
@@ -72,6 +72,7 @@
             nextPage: GET.and(bundle.$$BundleLinkUrl("next")).end(http),
             // For previous page, bundle.link.relation can either have 'previous' or 'prev' values
             prevPage: GET.and(bundle.$$BundleLinkUrl("previous")).and(bundle.$$BundleLinkUrl("prev")).end(http),
+            getBundleByUrl: GET.and(Path(":url")).end(http),
```

Comment:
Is this part of the FHIR spec?  What does it do?  If this is an API, shouldn't it be documented somewhere in this package?

Reply:
FHIR spec says, if it is a valid bundle of url it will return the bundle. Typically the valid url is obtained in link.relation and that is when it should be used. There are many coveats to document, but there is no API documentation any where, unless I create which is a task by itself. I am just going by the existing work flow here.